src/lib/logger/logger.mli

```diff
@@ -85,6 +85,8 @@ module Processor : sig
 
   val raw : ?log_level:Level.t -> unit -> t
 
+  val raw_structured_log_events : Structured_log_events.Set.t -> t
```

Comment:
to fix the JS build, you need to add fake implementations of these new methods to `logger/fake/logger.ml`